paper-slider.html

```diff
@@ -384,13 +394,25 @@
 
     _bardown: function(e) {
       e.preventDefault();
-      this._setTransiting(true);
+
       this._w = this.$.sliderBar.offsetWidth;
       var rect = this.$.sliderBar.getBoundingClientRect();
       var ratio = (e.detail.x - rect.left) / this._w;
+      var prevRatio = this.ratio;
+
+      this._setTransiting(true);
       this._positionKnob(ratio);
+
       this._expandJob = this.debounce(this._expandJob, this._expandKnob, 60);
```

Comment:
Not related to this change, but important: the return value of `this.debounce` is always `undefined`, so `this._expandJob` is being set to `undefined` here.